elasticsearch/Dockerfile

```diff
@@ -1,5 +1,6 @@
 # https://github.com/elastic/elasticsearch-docker
-FROM docker.elastic.co/elasticsearch/elasticsearch:5.6.3
+FROM docker.elastic.co/elasticsearch/elasticsearch-oss:6.0.0
 
 # Add your elasticsearch plugins setup here
 # Example: RUN elasticsearch-plugin install analysis-icu
+RUN elasticsearch-plugin install x-pack
```

Comment:
Why do you install X-pack just to disable it the config?

Reply:
Because otherwise ElasticSearch returns an exception during the bootstrap and the application can't be started. I sincerely don't know why and it sounds to me just a little bit strange.

Reply:
That's because you didn't delete `xpack.*` settings, so ES crashes due to unsupported options.